litgpt/deploy/serve.py

```diff
@@ -160,7 +160,8 @@ def setup(self, device: str):
             config = json.load(fp)
             chat_template = config.get("chat_template", None)
             if chat_template is None:
-                raise ValueError("chat_template not found in tokenizer config file.")
+                print("The tokenizer config does not contain chat_template, falling back to a default.")
```

Comment:
```suggestion
                print("Warning: The tokenizer config does not contain chat_template, falling back to a default.")
```

Ref for print being used in the repo: https://github.com/Lightning-AI/litgpt/blob/03778cc4bc67eaa7b4034ade9bec3177acb5c99d/litgpt/model.py#L66-L68

Reply:
Nah, it's not a warning, it's just informational